tests/e2e/TestConstants.ts

```diff
@@ -84,46 +84,16 @@ export const TestConstants = {
     TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: process.env.TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME || 'petclinic-dev-environment',
 
     /**
-     * Catalog tile operator name on OperatorHub UI.
+     * Using 'single-host' strategy, "false" by default.
      */
-    TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME: process.env.TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME || 'eclipse-che-preview',
-
-    /**
-     * Operator logo name by installation using OperatorHub
-     */
-    TS_SELENIUM_OPERATOR_LOGO_NAME: process.env.TS_SELENIUM_OPERATOR_LOGO_NAME || 'Eclipse Che',
-
-    /**
-     * Name of namespace created on OCP for installation CHE by using OperatorHub UI.
-     */
-    TS_SELENIUM_INSTALL_PROJECT_NAME: process.env.TS_SELENIUM_INSTALL_PROJECT_NAME || '',
-
-    /**
-     * Update Channel name on 'Create Operator Subscription' page on OCP, "nightly" or "stable".
-     */
-    TS_OCP_OPERATOR_UPDATE_CHANNEL: process.env.TS_OCP_OPERATOR_UPDATE_CHANNEL || 'nightly',
-
-    /**
-     * Value of TLS Support property in the 'Create Che Cluster' yaml using OperatorHub.
-     */
-    TS_SELENIUM_VALUE_TLS_SUPPORT: process.env.TS_SELENIUM_VALUE_TLS_SUPPORT || 'true',
-
-    /**
-     * Value of Self Sign Cert property in the 'Create Che Cluster' yaml using OperatorHub.
-     */
-    TS_SELENIUM_VALUE_SELF_SIGN_CERT: process.env.TS_SELENIUM_VALUE_SELF_SIGN_CERT || 'false',
```

Comment:
The same https://github.com/eclipse/che/pull/18714/files#r550222601

Reply:
Yes, it's the same which I described above.